spec/controllers/spree/orders_controller_spec.rb

```diff
@@ -48,8 +48,7 @@
             receive(:valid?).and_return(false)
           )
           allow_any_instance_of(Spree::LineItem).to(
-            receive_message_chain(:errors, :full_messages).
-              and_return(["Order population failed"])
+            receive_message_chain(:errors, :full_messages).and_return(["Order population failed"])
```

Comment:
why this change?

Reply:
Ouch, I forgot to remove it.
I was evaluating how many issues appear when we clean _rubocop_todo_.
Updating...

Reply:
Correction 😅 
That change is needed, it can be handled by Rubocop auto-correct.